### PR TITLE
update entitlements message

### PR DESCRIPTION
**Pull request by jennyf19**

### CODE REVIEW

File: core/src/Platforms/iOS/TokenCacheAccessor.cs [hunk 1]
```diff
@@ -288,7 +289,10 @@ private SecStatusCode Save(string account, string service, string generic, int t
             {
                 throw CoreExceptionFactory.Instance.GetClientException(
                 CoreErrorCodes.MissingEntitlements,
-                CoreErrorMessages.MissingEntitlements);
+                CoreErrorMessages.MissingEntitlements +
```
Comment (henrik-me) [inline core/src/Platforms/iOS/TokenCacheAccessor.cs]: >MissingEntitlements [](start = 34, length = 19)

update this one to take the parameter? or add a new message in CoreErrorMessages. #Closed

**Review by henrik-me — CHANGES_REQUESTED** [review]
:clock1:

File: core/src/Platforms/iOS/TokenCacheAccessor.cs (hunk 1 above)
Comment (jennyf19) [reply core/src/Platforms/iOS/TokenCacheAccessor.cs]: yes. fixed. #Closed

**Review by henrik-me — APPROVED** [review]
:shipit:

**Comment by henrik-me** [comment]
Looks great!

**Comment by henrik-me** [comment]
https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/633

**Pull request merged**